src/components/manager/runs/RunLastContent.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import { Table } from "antd";
+import { useSelector } from "react-redux";
+
+const LastIngestionTable = () => {
+    const runs = useSelector((state) => state.runs.items);
+
+    const [ingestions, setIngestions] = useState([]);
+
+    const processIngestions = (data) => {
+        const latestIngestions = data.reduce((acc, obj) => {
+            if (obj.state === "COMPLETE") {
+                const dataType = obj.details.request.tags.workflow_metadata.data_type;
+                const tableId = obj.details.request.tags.table_id;
+                // Skip the process if the tableId is undefined
+                if (tableId === undefined) {
+                    return acc;
+                }
+                const fileNameKey = Object.keys(obj.details.request.workflow_params)[0];
+                const filePath = obj.details.request.workflow_params[fileNameKey];
+                const fileName = filePath.split("/").pop();
+                const date = obj.details.run_log.end_time.split("T")[0];
```

Comment:
are you doing it this way vs using Date.parse in order to group by day rather than anything more specific? if so, i'd maybe add a comment saying why you are doing this & using string compare vs. using `Date.parse` as would be more expected for JS